simulation-ros/src/turtlebot2i/turtlebot2i_description/v-rep_model/README.md

```diff
@@ -0,0 +1,38 @@
+# Turtlebot2i V-rep model
+
+This folder contains the scene files e.g. **turtlebot2i.ttt** and Lua files for each component e.g. **turtlebot2i_lidar.lua**.
+
+The scene files are:
+
+1. mapir_lab_turtlebot2i.ttt
```

Comment:
I think this is marginally more helpful than just running 'ls'.

1. Which is the most recent file?
1. What is a one-line description of each scene file?